### ISSUE TITLE
Cannot use chain 43113 in evironment internal

### ISSUE THREAD

**Issue by smokeynotes** [CaskProtocol/cask-js-sdk#6]
I'm unable to use the Fuji chain in the Internal environment.

This code...
```
try {
    console.log(`Cask.currentChain()`)
    console.log(Cask.currentChain())

    let signer = null
    if (window && (window as any).ethereum) {
      const provider = new ethers.providers.Web3Provider((window as any).ethereum)
      signer = provider.getSigner()
    }
    console.log(`Cask.switchChain(${chainId}, ${await signer?.getAddress()})`)
    await Cask.switchChain(chainId, signer)
    console.log(Cask.currentChain())
  } catch (e: any) {
    console.dir(e)
}
```
outputs....
```
Cask.currentChain() 
80001
Cask.switchChain(43113, 0xD9cba6FCAdf21111B6a1BcED5FC466c47bac7f2F)
Error: Cannot use chain 43113 in evironment internal
```

even though this code https://github.com/CaskProtocol/cask-js-sdk/blob/main/src/core/EthersConnection.js#L104

should work since the vault has an address for the Fuji chain in env Internal https://github.com/CaskProtocol/cask-js-sdk/blob/main/src/core/deployments.js#L53

**Comment by masha256** [comment]
The version of the SDK you are using had a bug, fixed in 1.1.7.

**Comment by masha256** [comment]
Sorry, fixed in 1.1.8
